roles/collectd_config/templates/logfile.conf.j2

```diff
@@ -0,0 +1,10 @@
+# The plugin is loaded in collectd.conf
```

Comment:
You can use `{{ ansible_managed }}` in your template headers to automatically generate a header if you wanted.

Reply:
I could do that in all of them.
In this case, the LoadPlugin statement is in collectd.conf, rather than in logfile.conf, to force the logfile plugin to load first